### PR TITLE
Enable SiP for 5495

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4479

### CODE REVIEW

File: src/js/edu-benefits/5495/config/form.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ import fullNameUI from '../../../common/schemaform/definitions/fullName';
 
 import * as personId from '../../../common/schemaform/definitions/personId';
 
-import IntroductionPage from '../components/IntroductionPage';
+import IntroductionPage from '../components/SIPIntroductionPage';
```
Comment (jbalboni) [inline src/js/edu-benefits/5495/config/form.js]: I think you can just update the current intro page. It's easy enough to just revert this commit if we need to.

File: src/js/edu-benefits/5495/config/form.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ const formConfig = {
   trackingPrefix: 'edu-5495-',
   formId: '22-5495',
   version: 0,
-  disableSave: true,
+  // disableSave: true,
```
Comment (jbalboni) [inline src/js/edu-benefits/5495/config/form.js]: I had also added in `savedFormMessages`, based on what we did for pensions and hca.

**Review by jbalboni — APPROVED** [review]

**Review by cehsu — APPROVED** [review]
LGTM. No additional comments beyond what @jbalboni mentioned.

File: src/js/edu-benefits/5495/components/IntroductionPage.jsx [hunk 3]
```diff
@@ -60,26 +60,34 @@ class IntroductionPage extends React.Component {
...
-              afterText="»"/>
-          </div>
-        </div>
-        <div className="omb-info--container">
+        <SaveInProgressIntro
+          pageList={this.props.route.pageList}
+          messages={this.props.route.formConfig.savedFormMessages}
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}>
+          Please complete the 22-5495 form to apply for education benefits.
```
Comment (cehsu) [inline src/js/edu-benefits/5495/components/IntroductionPage.jsx]: It looks like we have inconsistent language here and in the 1990e, do we need to confirm the content for this across these forms?

Comment (jbalboni) [reply src/js/edu-benefits/5495/components/IntroductionPage.jsx]: Yeah, we probably should. I think this one can at least be changed to match the format in HCA/1990e/pension/burials. I think that language is from Peggy. We can create an issue for her to go back through them and confirm.

Comment (jbalboni) [reply src/js/edu-benefits/5495/components/IntroductionPage.jsx]: Yeah, we probably should. I don't think it needs to hold this up, but we need to have her look at them.

**Pull request merged**